api/v2/routes_lens.go

```diff
@@ -28,59 +27,59 @@ func (api *API) submitIndexRequest(c *gin.Context) {
 			Fail(c, err)
 			return
 		}
+		indexType = pb.IndexReq_IPLD
 	default:
 		Fail(c, errors.New(eh.InvalidObjectTypeError))
 		return
 	}
-	// send the index request
-	resp, err := api.lens.Index(context.Background(), &request.Index{
-		Type:       forms["object_type"],
-		Identifier: forms["object_identifier"],
-		// allow for optional reindexing
-		Reindex: c.PostForm("reindex") == "yes",
+
+	resp, err := api.lens.Index(context.Background(), &pb.IndexReq{
+		Type:    indexType,
+		Hash:    forms["object_identifier"],
+		Options: &pb.IndexReq_Options{
+			//	Reindex: c.PostForm("reindex") == "true",
```

Comment:
whats up here?

Reply:
this is temporarily disabled until we iron out the issue with reindexing that doesn't cause the queue to flush triggering the removal before attempting an index.

Reply:
see https://slack-redir.net/link?url=https%3A%2F%2Fgithub.com%2FRTradeLtd%2FLens%2Fpull%2F24%23issuecomment-466736622 - can't reproduce issue (removal is not a part of reindexing)